mmv1/products/vmwareengine/PrivateCloud.yaml

```diff
@@ -90,6 +91,16 @@ parameters:
     description: |
       The ID of the PrivateCloud.
 
+virtual_fields:
+  - !ruby/object:Api::Type::Integer
+    name: "deletion_delay_hours"
+    description: |
+      The number of hours to delay this request. You can set this value to an hour between 0 to 8, where setting it to 0 starts the deletion request immediately. If no value is set, a default value is set at the API Level.
```

Comment:
What is the API default value for deletion delay? Is it 0? How often do you think users will need to additionally set `send_deletion_delay_hours_if_zero`?

Reply:
Currently, the default delay at the API level is set to 3 hours. However, it might be modified in the near future. 

Users would need to set `send_deletion_delay_hours_if_zero` if they want to send `delay_hours=0` through the API call and delete the PC immediately. I have added this boolean field after a discussion with Stephen in a previous thread since terraform does not differentiate between "no value set" and "0 value set".

Reply:
Just noting that I do not like the need for `send_deletion_delay_hours_if_zero` but cannot think of another way to force send 0 only when the user has specified due to TF limitations. I am sure this was discussed previously

Reply:
Yes, [this](https://github.com/GoogleCloudPlatform/magic-modules/pull/10764#discussion_r1655305161) is the previous discussion thread for implementing this change. 

Another approach which I could think of was using `delay_hours` inside a nested object `deletion_parameters` but nested objects are not yet supported for `virtual_fields`.